io-engine/src/bin/io-engine.rs

```diff
@@ -92,10 +92,29 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
 }
 
 fn hugepage_get_nr(hugepage_path: &Path) -> (u32, u32) {
-    let nr_pages: u32 = sysfs::parse_value(hugepage_path, "nr_hugepages")
-        .expect("failed to read the number of pages");
-    let free_pages: u32 = sysfs::parse_value(hugepage_path, "free_hugepages")
-        .expect("failed to read the number of free pages");
+    let nr_pages: u32 = match sysfs::parse_value(hugepage_path, "nr_hugepages") {
+        Ok(c) => c,
+        Err(e) => {
+            log::error!(
```

Comment:
```suggestion
            error!(
```
To use the `tracing` instead of `log` crate.
Also, should this be a warning since we're defaulting to 0 and check the `PAGES_NEEDED` anyway?

Reply:
Makes sense to me 👍

Reply:
Updated to a `warn!`